### DISCUSSION TITLE
terrain flicker error

### DISCUSSION THREAD

**Discussion by aaaaaaaaaaaaaaasdasd** [FabricMC/fabric-api#3893] [Player support]
### Description

Every time I break a block the terrain flashes, it also happens when another player breaks a block my terrain flashes, I only use sodium
I have this error with all versions of fabric

**Comment by Poopooracoocoo** [comment]
Logs are not something you can just skip. They're always generated. 

You likely have outdated Intel graphics drivers. 
https://github.com/CaffeineMC/sodium-fabric/wiki/Driver-Compatibility#gen11-graphics---ice-lake-2019
Update them here: https://www.intel.com/content/www/us/en/download/776137/intel-7th-10th-gen-processor-graphics-windows.html

**Reply by aaaaaaaaaaaaaaasdasd** [reply]
My drivers are updated and so is the fabric and sodium version. I have a friend with an Nvidia GPU and the same thing happens to him too.

**Reply by Poopooracoocoo** [reply]
Please share a log. It's the `.minecraft/logs/latest.log` file. File extensions are disabled by default so it'll be called "latest". Upload it as a file here.

**Reply by aaaaaaaaaaaaaaasdasd** [reply]
The file you are referring to only has information from a game made in version 1.8, in that version I do not have the error

**Reply by Poopooracoocoo** [reply]
Start the problematic installation again, then share the log.
